example/tutorials/code/tf_iris.ipynb

```diff
@@ -0,0 +1,387 @@
+      "  np_resource = np.dtype([(\"resource\", np.ubyte, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  _np_qint8 = np.dtype([(\"qint8\", np.int8, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  _np_quint8 = np.dtype([(\"quint8\", np.uint8, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  _np_qint16 = np.dtype([(\"qint16\", np.int16, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  _np_quint16 = np.dtype([(\"quint16\", np.uint16, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  _np_qint32 = np.dtype([(\"qint32\", np.int32, 1)])\n",
+      "/home/juzix/anaconda3/envs/rosetta/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
+      "  np_resource = np.dtype([(\"resource\", np.ubyte, 1)])\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import math\n",
+    "import matplotlib.pyplot as plt\n",
+    "import tensorflow as tf\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "np.set_printoptions(suppress=True)\n",
+    "\n",
+    "os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'\n",
+    "\n",
+    "np.random.seed(0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "looking-watts",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# #############################################################################\n",
+    "# # This method of data loading is not suitable for static graph in TF1.14, \n",
```

Comment:
I don't understand this, any more explanation?

Reply:
These lines are prepared for future Rosetta which may support dynamic graph. The dynamic graph can tell users more about what they are doing in data and model dimensions. Now many popular python libs provide with APIs to load classic datasets like Iris, and the up-to-date tutorials of loading Iris in TF is eager mode loading.
Many teachers in their ML and DL courses recommend students to deduce the flows of tensor on scratch paper. Noobies will refer to latest official tutorials and they use eager mode data loading as TF official say. These line can remind noobies of avoiding them concat code and wheels roughly from others.